docs/guides/development/Limits.md

```diff
@@ -19,10 +25,45 @@ To understand the basic limits of your account, check out [Pricing](https://www.
 | Billing cycle API requests<br><br>Bulk API requests are not added to the billing cycle limit. | Free plan: 500,<br>Startup plan: 10,000,<br>Growth plan: 30,000,<br>Professional plan: 100,000 |
 | Client-side requests (e.g., validate) from a single IP address | 5 requests per 5 seconds |
 
+To understand the basic limits of your account, check out [Pricing](https://www.voucherify.io/pricing) and [Fair Use Policy](https://www.voucherify.io/legal/fair-use-policy).
+
+## Notifications
+
+You can receive notifications in your app and through email when you use a certain percentage of your redemptions, API calls, or bulk API calls. The account owner can configure these thresholds and notifications in the `Project settings > API Usage` tab.
+To set this up, the account owner has two options:
+
+1. Go to the Bell Icon at the top of your Dashboard, then click on Notifications Center and select Account Settings.
+
+2. Alternatively, click the pencil icon under one of the charts. This will take you directly to the API usage notification settings.
+
+To get email notifications, follow these steps:
+
+1. Visit Notifications Center > Account Settings.
+
+2. Click on Show details next to the notifications you want to activate.
+
+3. Turn on the switch next to Email.
+
+4. Set the percentage usage limit that will trigger the notification.
+
+5. Click Add email to specify the email address where you want to receive notifications.
+
+6. Save your settings.
+
+For in-app notifications, simply toggle the switch next to "In app."
+
+Notifications for webhooks usage will be sent if the usage exceeds the limit for an hour, even though they are counted in daily periods. It is a measure to inform you in advance about any suspicious user activity.
```

Comment:
Drugie zdanie bym poprawił na:
`As a result, you will be informed in advance about any suspicious user activity.`